Backend/Remora.Discord.API.Abstractions/API/Rest/IDiscordRestGuildAPI.cs

```diff
@@ -335,6 +335,20 @@ Task<Result> ModifyGuildMemberAsync
             CancellationToken ct = default
         );
 
+        /// <summary>
+        /// Modifies the current member.
+        /// </summary>
+        /// <param name="guildID">The ID of the guild.</param>
+        /// <param name="nickname">The new nickname.</param>
+        /// <param name="ct">The cancellation token for this operation.</param>
+        /// <returns>A modification result which may or may not have succeeded, containing the updated member.</returns>
+        Task<Result<IGuildMember>> ModifyCurrentMemberAsync
```

Comment:
It seems very unusual that this endpoint would not support the `X-Audit-Log-Reason` header. I wonder if this is a mistake on behalf of the docs.

Reply:
Testing shows that the endpoint does indeed support the header, and the docs are incorrect.

Reply:
Probably should open an issue on the docs repo,or I can when I get home from school later.

Reply:
I was planning on opening a PR to fix it but haven't gotten around to it yet. If you'd like to do that it'd be great!

Reply:
@n-Ultima I've gone through and made the PR 👍

Reply:
I saw when I was at school earlier, let's hope it get's merged easily :)